packages/styles/notice.css

```diff
@@ -33,6 +33,22 @@
   font-size: var(--text-size-smaller);
 }
 
+.Notice--default {
+  display: grid;
+  padding: var(--space-large);
+  grid-template-columns: auto 1fr;
+  gap: var(--space-small);
+  align-items: start;
+}
+
+.Notice--condensed {
```

Comment:
We shouldn't duplicate properties here that are defined by all variants. If we change `.Notice--default` to be applied across all notices, we can just have properties that are specific to condensed. Primarily the number of columns and reduced padding.